### ISSUE TITLE
Option to choose which drive is protected

### ISSUE THREAD

**Issue by ikem-krueger** [yyounan/MBRFilter#8]
I like to have an option to choose which drive is protected by MBRFilter.

I like to have only my Windows drive protected by MBRFilter.

**Comment by mcprat** [comment]
I agree although I don't know if it is possible without major changes to the program. It seems to make the protection of 0 sectors something like a system policy. I thought that it was the original purpose to just do this for the OS drive and I have been confused about why I have been unable to format flash drives or create bootable partitions...
